### DISCUSSION TITLE
Ignite da Rocketseat vale a pena?

### DISCUSSION THREAD

**Discussion by Carrilhos** [frontendbr/forum#2070] [Perguntas e Respostas]
Alguém já fez o ignite da Rocketseat? estava pensando em fazer a trilha de react e queria saber mais sobre o conteúdo trabalhado

**Comment by IagoNascimentocode** [comment]
Sim, com certeza.

**Comment by helenapaixao** [comment]
Muiiito, recomendo!!

Em sex, 30 de abr de 2021 01:01, Iago Nascimento ***@***.***>
escreveu:

**Comment by RicardoWEBSiTE** [comment]
SUPER vale a pena, vc consegue aprender muita coisa desde o básico ao
avançado.


Em qua., 28 de abr. de 2021 às 14:17, Gabriel Oliveira Carrilhos <
***@***.***> escreveu:

**Comment by theandersonn** [comment]
@Carrilhos dá uma olhada nessa thread https://github.com/frontendbr/forum/discussions/1512, as opniões vão lhe ajudar visto que o Ignite é a evolução do goStack.

**Comment by ghost** [comment]
Pessoal,

Permita-me discordar de um ponto. Na minha opinião o curso é muito bom sim,
utiliza tecnologias novas e é bem completo. No entanto, se você não possuir
uma base boa de JS e ir direto para o Ignite poderá ter certa dificuldade.

Se você não tem base de JS meu conselho é fazer os cursos básicos que eles
oferecem gratuitamente e só depois investir no curso pago.

Em sex., 30 de abr. de 2021 às 11:01, Ricardo Nascimento <
***@***.***> escreveu:

> SUPER vale a pena, vc consegue aprender muita coisa desde o básico ao
> avançado.
>
>
> Em qua., 28 de abr. de 2021 às 14:17, Gabriel Oliveira Carrilhos <
> ***@***.***> escreveu:
>
> > Alguém já fez o ignite da Rocketseat? estava pensando em fazer a trilha
> de
> > react e queria saber mais sobre o conteúdo trabalhado
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/frontendbr/forum/discussions/2070>, or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAHDKUWV6SMS4HABQK6P4GTTLA7MJANCNFSM43XP42LA
> >
> > .
> >
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/discussions/2070#discussioncomment-679676>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJZQ7EUT75OV3HD36I4NAXTTLKZ4FANCNFSM43XP42LA>
> .
>


-- 

Atenciosamente,



*Marcus Vinicius de Jesus Azevedo*

**Reply by Cclaudia13** [reply]
Ola.
Vejo esses comentarios aqui em outubro de 2023 e fico na duvida se em 2021, a parte gratuita que oferecem, tinham os videos focados. Estou tentando seguir as aulas do excelente Mike porem sem foco na tela que ele explica fica praticamente impossivel. Uma grande pena pois queria tanto fazer o basico pois gosto do Mike e adorei o projeto inicial. Escrevo e comento as aulas mas ninguem responde.

**Comment by wineerx** [comment]
Sim, bastante

**Comment by wilsonneto-swe** [comment]
@Carrilhos fiz um repo com alguns cursos que fiz e acheo muito bons, Rocketseat com certeza está ali, mas é o mais carinho da lista rs TTodos excelentes:

https://github.com/wilsonneto-dev/Melhores_Cursos_Online
